index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <header class="header">
+        <div class="header-contain">
+            <div class="logo">My Logo</div>
+            <div>
+                <img class="vector" src="./assets/img/Union.png" alt="">
+                
+            </div>
```

Comment:
- [ ] For [Semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) and best user experience I would suggest to use `<nav>` tag on the hamburger. This helps on easy accessibility of the webpage by the screen readers. 👏